test/README.md

```diff
@@ -87,12 +87,11 @@ go test -v -tags=e2e -count=1 ./test -run ^TestEventListener
 
 There are no yaml tests at the moment. These tests will be created under
 [Issue #11](https://github.com/tektoncd/triggers/issues/11).
```

Comment:
Thanks for adding these instructions to the test README. However, can you please delete these two above lines now that we have the yaml tests 😄

Reply:
Whoops. Will do.